salt/modules/cryptdev.py

```diff
@@ -190,16 +193,16 @@ def rm_crypttab(name, config='/etc/crypttab'):
                     lines.append(line)
 
     except (IOError, OSError) as exc:
-        msg = "Couldn't read from {0}: {1}"
-        raise CommandExecutionError(msg.format(config, str(exc)))
+        msg = 'Could not read from {0}: {1}'
+        raise CommandExecutionError(msg.format(config, six.text_type(exc)))
 
     if modified:
         try:
             with salt.utils.files.fopen(config, 'w+') as ofile:
-                ofile.writelines(lines)
+                ofile.writelines((salt.utils.stringutils.to_str(line) for line in lines))
         except (IOError, OSError) as exc:
-            msg = "Couldn't write to {0}: {1}"
-            raise CommandExecutionError(msg.format(config, str(exc)))
+            msg = 'Could not write to {0}: {1}'
+            raise CommandExecutionError(msg.format(config, six.text_type(exc)))
```

Comment:
We don't need `six.text_type` here, because `.format()` is already going to run `unicode(exc)` under the hood when formatting. (We didn't need `str()` there before, either).